packages/renderer/src/lib/preferences/PreferencesRegistriesRendering.svelte

```diff
@@ -52,49 +52,54 @@ function toggleRegistryModal(): void {
-                type="button">
-                <i class="fas fa-pen" aria-hidden="true"></i>
-              </button>
-              <button
-                on:click="{() => removeRegistry(registry)}"
-                title="Remove registry"
-                class="inline-block text-gray-500 dark:text-gray-500 hover:bg-gray-100 dark:hover:bg-gray-100 focus:outline-none rounded-lg text-sm p-1.5"
-                type="button">
-                <i class="fas fa-times" aria-hidden="true"></i>
-              </button>
-            </div>
-            <div class="px-2">
-              <h2 class="text-xl font-bold font-bol text-gray-800">{registry.serverUrl}</h2>
-              <p class="text-small text-gray-600">{registry.username}</p>
-              <p class="w-full font-extralight text-right text-xs text-gray-800">{registry.source}</p>
+        <div class="pf-l-flex pf-m-column pf-m-align-self-flex-start pb-4">
+          <div class="pf-l-flex__item">
+            <div class="w-full p-4 pf-u-flex-tile bg-zinc-700 rounded shadow-md">
+              <div class="flex justify-end flex-wrap">
+                <button
+                  on:click="{() => {
+                    editRegistry(registry);
+                  }}"
+                  class="inline-block text-gray-100 dark:text-gray-500 hover:bg-gray-600 dark:hover:bg-gray-600 focus:outline-none rounded-lg text-sm px-2 py-1"
+                  title="Edit registry"
+                  type="button">
+                  <i class="fas fa-pen" aria-hidden="true"></i>
+                </button>
+                <button
+                  on:click="{() => removeRegistry(registry)}"
+                  title="Remove registry"
+                  class="inline-block text-gray-100 dark:text-gray-500 hover:bg-gray-600 dark:hover:bg-gray-600 focus:outline-none rounded-lg text-sm px-2 py-1"
+                  type="button">
+                  <i class="fas fa-times" aria-hidden="true"></i>
+                </button>
+              </div>
+              <div class="px-2">
+                <h2 class="text-xl font-bold text-gray-100 pf-u-text-truncate">{registry.serverUrl}</h2>
+                <p class="text-small text-gray-100">{registry.username}</p>
+                <p class="w-full font-extralight text-right text-xs text-gray-100">{registry.source}</p>
+              </div>
             </div>
           </div>
         </div>
       {/each}
-      <div class="w-full p-4 md:w-60 md:h-60">
-        <div
-          on:click="{() => {
-            createRegistry();
-          }}"
-          class="p-6 bg-white rounded shadow-md  hover:bg-gray-100 dark:hover:bg-gray-100 hover:cursor-pointer">
-          <div class="px-2 text-center">
-            <div class="text-3xl text-purple-800 fas fa-plus-circle" aria-hidden="true"></div>
-            <h2 class="text-xl font-bold font-bol text-purple-800">Add registry</h2>
+      <div class="pf-l-flex pf-m-column pf-m-align-self-flex-start">
```

Comment:
Hello, I'm wondering why you're using patterngly flex component and not the tailwind like almost everywhere ?
The Sae for alignment

Reply:
Updated with using Tailwindcss